lang/go/tools/pkgs/colorize/colorize.go

```diff
@@ -127,11 +127,12 @@ func SetDefaultFlags(f string) {
 }
 
 // Colorize - Add color to the value passed, w size can be 0, 1 or 2
-//   w[0] is the width and w[1] is precision or a float value if present
-//   w[1] is not present then use default colorInfo.floatPrecision
-//   w[2] is the foreground color
-//   w[3] is the background color
-//   w[4] is the attribute of the color
+//
+//	w[0] is the width and w[1] is precision or a float value if present
```

Comment:
second time, probably a `gofmt` thing?

Reply:
Yes, I noticed these and I believed if was the formatter causing the changes.